types/oracledb/index.d.ts

```diff
@@ -1,4 +1,4 @@
-// Type definitions for oracledb v3.1.2
+// Type definitions for oracledb v4.0.0
```

Comment:
Could you please enable `dt-header` and change this from `v4.0.0` to `4.0`? We maintain our own patch version automatically. You should just need to remove `"dt-header": false` from `tslint.json`.

Also, if you expect anyone to need to update types for 3.0, you should [retain the old version in a subfolder](https://github.com/DefinitelyTyped/DefinitelyTyped#if-a-library-is-updated-to-a-new-major-version-with-breaking-changes-how-should-i-update-its-type-declaration-package)